source/includes/edge/edgectl-status.rst

```diff
@@ -0,0 +1,88 @@
+When the Edge Server instance is running, you can see the status of the 
+instance with:
+
+.. code-block:: shell
+
+   edgectl status
+
+This gives you information about the state of the Edge Server containers, and
+a detailed status for the Sync server with a breakdown of important 
+information. It may look similar to:
+
+.. code-block:: json
+   :copyable: false
+
+   {
+      "mongoContainerStatus": {
+         "message": "running"
+      },
+      "mongoDBStatus": {
+         "message": "ready"
+      },
+      "syncServerContainerStatus": {
+         "message": "running"
+      },
+      "syncServerStatus": {
+         "message": "ACTIVE"
+      },
+      "syncServerDetailedStatus": {
+         "version": "v0.22.3",
+         "status": "ACTIVE",
+         "cloud_connected": true,
+         "num_local_clients": 0,
+         "query": {
+            "Item": "truepredicate"
+         }
+      }
+   }
+
+The ``syncServerDetailedStatus`` contains information about the state of the
+Edge Server instance. This includes details about its connection to
+Atlas, and the clients connected to it, as follows:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 70
+
+   * - Field name
+     - Type
+     - Value
+
+   * - ``version``
+     - String
+     - The current version of the Edge Server code that the instance is 
+       running. If the Edge Server instance falls more than one major version
+       behind the latest available version, it is unable to connect to Atlas
+       and must be wiped and updated. For more information, refer to 
+       :ref:`upgrade-edge-server-version`.
+
+   * - ``status``
+     - String
+     - The current status of the Edge Server instance. Possible values include:
+       ``"ACTIVE"``, ``"FAILED"``. 
```

Comment:
there's also BOOTSTRAPPING, which happens when the edge server is making its initial download and not yet ready to use